README.markdown

```diff
@@ -315,7 +319,11 @@ Location of the step tickers file on the managed system. Default value: varies b
 
 ####`step_tickers_template`
 
-Location of the step tickers template file. Default value: varies by operating system
+Location of the step tickers ERB template file. Default value: varies by operating system. Validation error will be thrown if this is specified as well as the `step_tickers_epp` param.
+
+####`step_tickers_epp`
+
+Location of the step tickers EPP template file. Default value: varies by operating system. Validation error will be thrown if this is specified as well as the `step_tickers_template` param.
```

Comment:
This varies by OS in that in Redhat.yaml a step_tickers_file value is supplied whereas in every other OS it is not. So on redhat the default behaviour is to use the default step-tickers.epp template file whereas on other OS's by default no step-tickers file is used.